src/wallet.ts

```diff
@@ -8,6 +8,7 @@ import {
   confidential,
   Transaction,
 } from 'liquidjs-lib';
+import { Output } from 'liquidjs-lib/types/transaction';
```

Comment:
Personally I prefer to use `TxOutput` from liquidjs directly. It avoids large import path such as /types/transaction.